ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -195,6 +198,25 @@ public ConsistencyCheckerTool(ClusterMap clusterMap, StoreKeyFactory storeKeyFac
     return new Pair<>(success, resultsByReplica.getSecond());
   }
 
+  /**
+   * Will examine the index files in the File replicas for store keys
+   * and will then convert these store keys and return the
+   * conversion map
+   * @param replicas replicas that should contain index files that will be
+   *                 parsed
+   * @return conversion map of old store keys and new store keys
+   * @throws Exception
+   */
+  public Map<StoreKey, StoreKey> convertBlobIds(File[] replicas) throws Exception {
```

Comment:
why is this public?

Reply:
Where is this function used?

Reply:
Used for caching store converter values from a set of replicas.  For instance, can be useful in case one wants to give a message transformer constructor a cache ahead of time instead of having to invoke the converter for messages one at a time.

Reply:
That kind of API doesn't belong here though. Whoever needs the cache can generate it OR we can have another helper do it - the `ConsistencyCheckerTool` shouldn't be providing those services